virttest/qemu_vm.py

```diff
@@ -2351,6 +2351,8 @@ def sort_key(dev):
         virtio_pci_devices = ["virtio-net-pci", "virtio-blk-pci",
                               "virtio-scsi-pci", "virtio-balloon-pci",
                               "virtio-serial-pci", "virtio-rng-pci"]
+                              "virtio-keyboard-pci", "virtio-mouse-pci",
```

Comment:
repeated ']' in line 2353, need to change to ','

Reply:
Yes, modified, thanks